src/platform/x11.rs

```diff
@@ -138,9 +138,16 @@ impl EventLoopBuilderExtX11 for EventLoopBuilder {
 /// Additional methods on [`Window`] that are specific to X11.
 ///
 /// [`Window`]: crate::window::Window
-pub trait WindowExtX11 {}
+pub trait WindowExtX11 {
+    fn set_override_redirect(&self, value: bool);
+}
 
-impl WindowExtX11 for dyn CoreWindow {}
+impl WindowExtX11 for dyn CoreWindow {
+    fn set_override_redirect(&self, value: bool) {
+        let window = self.as_any().downcast_ref::<crate::platform_impl::Window>().unwrap();
```

Comment:
The downcast should be straight to `x11` type, I'm not sure the type you're using even exists.

Reply:
thanks, changed the downcast to `platform_impl::x11::window::Window`